bot/exts/evergreen/trivia_quiz.py

```diff
@@ -559,7 +640,7 @@ def category_embed(self) -> discord.Embed:
     @staticmethod
     async def send_answer(
         channel: discord.TextChannel,
-        answers: List[str],
+        answers: t.List[str],
```

Comment:
This will conflict with the current codebase and we should use `list` since we're on 3.9.